doc/Sidecar/kubernetes_security_guide.md

```diff
@@ -20,49 +21,48 @@ To view a copy of this license, visit <https://creativecommons.org/licenses/by-s
-* [3 Handling passwords and secrets](#3-handling-passwords-and-secrets)
-  * [3.1 Secrets as volume](#31-secrets-as-volume)
-  * [3.2 Secrets as environment variables](#32-secrets-as-environment-variables)
-* [4 User accounts](#4-user-accounts)
-  * [4.1 Create a kubeconfig](#41-create-a-kubeconfig)
-  * [4.2 Grant Cluster access to users](#42-grant-cluster-access-to-users)
-  * [4.3 Restrict namespace access to a cluster](#43-restrict-namespace-access-to-a-cluster)
-  * [4.4 Kubernetes Dashboard](#44-kubernetes-dashboard)
-* [5 Network policies](#5-network-policies)
-  * [5.1 Create Network policies](#51-create-network-policies)
-* [6 Pod Security Policies](#6-pod-security-policies)
-  * [6.1 Pod Security Policies in AWS EKS](#61-pod-security-policies-in-aws-eks)
-  * [6.2 Creating a Pod Security Policy](#62-creating-a-pod-security-policy)
-* [7 Assign Resources to Containers and Pods](#7-assign-resources-to-containers-and-pods)
-* [8 Monitoring](#8-monitoring)
-* [9 Backups](#9-backups)
-* [10 Message logs](#10-message-logs)
-* [11 Horizontal Pod Autoscaler best practices](#11-horizontal-pod-autoscaler-best-practices)
+- [License](#license)
+- [Table of contents](#table-of-contents)
+- [1 Introduction](#1-introduction)
+  - [1.1 Target Audience](#11-target-audience)
+- [2 Reference Data](#2-reference-data)
+- [3 Handling passwords and secrets](#3-handling-passwords-and-secrets)
+  - [3.1 Secrets as volume](#31-secrets-as-volume)
+  - [3.2 Secrets as environment variables](#32-secrets-as-environment-variables)
+- [4 User accounts](#4-user-accounts)
+  - [4.1 Create a kubeconfig](#41-create-a-kubeconfig)
+    - [4.1.2 Create a kubeconfig for AWS EKS](#412-create-a-kubeconfig-for-aws-eks)
+    - [4.1.2 Create a kubeconfig for Azure AKS](#412-create-a-kubeconfig-for-azure-aks)
+    - [4.1.3 Test the configuration](#413-test-the-configuration)
+  - [4.2 Grant cluster access to users](#42-grant-cluster-access-to-users)
+  - [4.3 Restrict namespace access to a cluster](#43-restrict-namespace-access-to-a-cluster)
+  - [4.4 Kubernetes Dashboard](#44-kubernetes-dashboard)
+- [5 Network policies](#5-network-policies)
+  - [5.1 Create Network policies](#51-create-network-policies)
+- [6 Pod Security Admission](#6-pod-security-admission)
+- [7 Assign Resources to Containers and Pods](#7-assign-resources-to-containers-and-pods)
+- [8 Monitoring](#8-monitoring)
+- [9 Backups](#9-backups)
+- [10 Message logs](#10-message-logs)
+- [11 Horizontal Pod Autoscaler best practices](#11-horizontal-pod-autoscaler-best-practices)
 
 ## 1 Introduction
 
 ### 1.1 Target Audience
 
-This User Guide is meant for X-Road Security Server system administrators responsible for installing and using X-Road Security Server Sidecar in AWS EKS environment.
+This User Guide is meant for X-Road Security Server system administrators responsible for installing and using X-Road Security Server Sidecar in AWS EKS or Azure Kubernetes Service (AKS) environment.
```

Comment:
In AWS documentation, EKS is referred to as Amazon EKS instead of AWS EKS. 

So, I will change to:
Amazon Elastic Kubernetes Service (Amazon EKS)
instead of:
AWS Elastic Kubernetes Service (EKS)


(also elsewhere in documentation)